### ISSUE TITLE
Text in like button

### ISSUE THREAD

**Issue by RonnyG2121** [nunocoracao/blowfish#2605]
Hello. I need to modify the like button so that when I want to like or dislike a post, the button updates the text displayed, because the screen reader doesn't read the emoji. Also, the views and likes section only shows the number and doesn't say "likes" or "views."
